src/$.js

```diff
@@ -94,7 +94,7 @@
 
                     // if the selector is an ID
                     // select the matching element and add it to the elements array
-                    if (selector.match(/^\#[^\s]+$/)) elements.push(parent.getElementById(selector.substr(1)));
```

Comment:
the selector is supposed to be an ID. in this case, `getElementById` is is faster than `querySelector`

Reply:
Yes, but only document has getElementById , other dom elements have not!

Reply:
change parent to document, is also okay, but when having 2 same ID;s in the document .... the parent is more usefull, and compatible with jquery